soccer/gameplay/plays/offense/distraction.py

```diff
@@ -0,0 +1,172 @@
+import main
+import robocup
+import behavior
+import constants
+import enum
+import standard_play
+import evaluation
+import tactics.coordinated_pass
+import skills.move
+import skills.capture
+
+
+class distraction(standard_play.StandardPlay):
+
+    class State (enum.Enum):
+        setup = 1, 'capture ball and move distractor and striker into position'
+        optional_adjustment = 2, 'setup a pass to the center right'
+        center_pass = 3, 'pass to the center right of the field'
+        passing = 4, 'pass from first capture to distractor'
+        cross = 5, 'pass from distractor to striker'
+        shoot = 6, 'shoot at goal'
+
+    def __init__(self):
+        super().__init__(continuous=False)  
+
+        self.ball_is_far = main.ball().pos.y < (0.4*constants.Field.Length)
+        self.distraction_point = robocup.Point(0.40*constants.Field.Width, 0.95*constants.Field.Length) #the first distraction point
+        self.distraction_recieve_pass_point = robocup.Point(0.40*constants.Field.Width, 0.8*constants.Field.Length) #the second distraction point
+        self.striker_point = robocup.Point(-0.40*constants.Field.Width, 0.9*constants.Field.Length) #striker's position
+        self.center = robocup.Point(0.5*constants.Field.Width,0.5*constants.Field.Length) #center of field position, used if ball is far
+        self.distract_box = robocup.Rect(
+            robocup.Point(0.33*constants.Field.Width/2,
+                (7/9)*constants.Field.Length),
+            robocup.Point(constants.Field.Width, 
+                constants.Field.Length))
+        self.striker_box = robocup.Rect(
+            robocup.Point(-0.33*constants.Field.Width/2,
+                (7/9)*constants.Field.Length),
+            robocup.Point(-constants.Field.Width, 
+                constants.Field.Length))
```

Comment:
I would try to format this to be cleaner. Lining up the x,y in the point args should be good enough, maybe a little whitespace